src/frequenz/channels/experimental/__init__.py

```diff
@@ -0,0 +1,16 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Experimental channel primitives.
+
+This package contains experimental channel primitives that are not yet
+considered stable. They are subject to change without notice.
```

Comment:
Maybe like this to make it more scary?

```suggestion
Warning:
    This package contains experimental channel primitives that are not yet
    considered stable. They are subject to change or removal without
    notice even in minor updates.
```

Reply:
Done with slightly different wording.